about.html

```diff
@@ -0,0 +1,106 @@
+                <li><a class="nav-link" href="#">Program</a></li>
+                <li><a class="nav-link" href="#">Join</a></li>
+                <li><a class="nav-link" href="#">Sponsor</a></li>
+                <i class="fa-solid fa-xmark"></i>
+              </ul>
+              <Span>Special Seat</Span>
+            </nav>
+            </div>
+        </header>
+        <div class="begin-about">
+                <div class="intro-about">
+                    <p>"Hello HennaHub Globe!"</p>
+                    <h1 class="intro-trans-anim">HENNAHUBUK'S CREATIVE DESIGN APPRENTICE SUMMIT <br>2023</h1>
+                </div>
+                <div>
+                    <p class="about-text">
+                       @ HennaHub, we offer henna tattoo of any type for any occasion with over 20 years experience, glitter tattoo and face painting. We also offer henna designs using body paint, patch test of 24 to 48 hours is mandatory before full henna application incase of allergies and sensitive skin to 100% plant product. The henna types we use are: 
+                       <br> Natural red henna and Natural black jagua henna 
+                    </p>
+                </div>
+                <div class="contact-us">
+                    Please Contact us per Email, for further enquiry about HennaHubuk apprentice summit 2023 <br>
+                    <span>hennahubuk@gmail.com</span>
+                  </div>
+                </div>
+                <div class="mid">
+                    <div class="mid-text">
+                      <h4>HennaHub Uk Mehndi Designs and Creativity Logo</h4>
+                      <hr>
+                      <p>The Hennahub Uk Apprentice Summit was inspired by the passion and designs of the trainees graduating in December 2023</p>
+                    </div>
+                    <div class="mid-image">
+                      <img src="./images/HennaHubMid.jpeg" alt="concert logo" class="img-mid">
+                    </div>
+                  </div>
+                  <div class="past">
+                    <div class="past-text">
+                      <h3>See the past exhibition summits</h3>
+                      <hr>
+                      <p>Take a look at previous HennaHubUK exhibitions in Glasgow Uk</p>
+                    </div>
+                    <div class="collage-exhibit-section">
+                      <div class="exhibit-img">
+                        <div class="exhibit-inner">
+                          <h3>2022</h3>
+                          <p>HennaHub 2022 exhibition display in Glasgow</p>
+                        </div>
+                      </div>
+                      <div class="exhibit-img1">
+                        <div class="exhibit-inner">
+                          <h3>2021</h3>
+                          <p>HennaHub 2021 exhibition display in Glasgow</p>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
```

Comment:
- [ ] Same here as pointed earlier :point_up_2: Kindly wrap this with a `section` tag.